samples/sample-ec2-java-app-codedeploy/scripts/start.sh

```diff
@@ -2,4 +2,4 @@
 
 cd /home/ec2-user/server
 sudo /usr/bin/java -jar -Dserver.port=80 \
-  *.jar > /dev/null 2> /dev/null < /dev/null &
+  *.jar >/dev/null 2>/dev/null </dev/null &
```

Comment:
Might be me, but personally I like the space after the `>` sign.
It makes things more readable I think. What are your thoughts?
And can we change this setting?

A bit surprising that it does enforce spaces around `|`.

Reply:
I agree with you. Adding `space_redirects = true` to editorconf to fix this.